cmd/oras/root/attach.go

```diff
@@ -149,8 +154,9 @@ func runAttach(ctx context.Context, opts attachOptions) error {
 		ManifestAnnotations: annotations[option.AnnotationManifest],
 		Layers:              descs,
 	}
+
 	pack := func() (ocispec.Descriptor, error) {
-		return oras.PackManifest(ctx, store, oras.PackManifestVersion1_1_RC4, opts.artifactType, packOpts)
+		return oras.PackManifest(ctx, store, opts.PackVersion, opts.artifactType, packOpts)
```

Comment:
We can do something like
```go
switch opts.PackVersion {
    case ImageSpecV1_0:
        return ocispec.Descriptor{}, errors.New("image-spec v1.0 is not supported")
    case PackManifestVersion1_1_RC2:
        return oras.Pack(ctx, store, opts.artifactType, packOpts.Layers, oras.PackOptions{
            Subject:             packOpts.Subject,
            ManifestAnnotations: packOpts.ManifestAnnotations,
            PackImageManifest:   true,
        })
    default:
        return oras.PackManifest(ctx, store, opts.PackVersion, opts.artifactType, packOpts)
}
```

Reply:
We certainly can, but that still requires a change in oras-go and a dep bump in oras. I figured we could keep the details out of the client with the initial proposal, but if we'd rather have it obvious and in the client, I can make this change.